<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Importer</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_main">
    <item>
     <widget class="QLabel" name="import_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="maya">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>maya</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>631</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="maya_layout">
         <item>
          <layout class="QHBoxLayout" name="maya_general_layout">
           <item>
            <widget class="QLabel" name="maya_general_label">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>General options</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="maya_general_line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="maya_group_check">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="maya_remove_check">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Remove duplicate shading networks</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="maya_ref_layout">
           <item>
            <widget class="QLabel" name="maya_ref_label">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Referencing options</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="maya_ref_line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="maya_preserver_check">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Preserve references</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="maya_preserve_combo">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Load saved reference load state</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Load all references</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Load top-level references only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Load no references</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="maya_nm_layout">
           <item>
            <widget class="QLabel" name="maya_nm_label">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Namespace options</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="maya_nm_line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="maya_namespaces_list">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="currentRow">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>:(root)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="maya_merge_layout">
           <item>
            <widget class="QRadioButton" name="maya_merge_radio">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Merge into selected namespace and rename incoming objects that match</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="maya_parentnm_layout">
           <item>
            <widget class="QRadioButton" name="maya_parentnm_radio">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Use selected namespace as parent and add new namespace string :</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="maya_parentnm_entry">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="maya_filename_layout">
           <item>
            <widget class="QRadioButton" name="maya_filename_radio">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Use selected namespace as parent and add new namespace (file name)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="alembic">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>alembic</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>631</width>
          <height>281</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="alembic_layout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="alembic_general_layout">
           <item>
            <widget class="QLabel" name="alembic_general_label">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>General options</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="alembic_general_line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="alembic_importroot_radio">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Import to scene root</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="alembic_importcurrent_radio">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Import under current selection</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="alembic_groupbox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="title">
            <string/>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>381</width>
              <height>94</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="alembic_groupbox_layout">
             <item>
              <widget class="QRadioButton" name="alembic_add_radio">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Add                                                                                       </string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="alembic_merge_radio">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Merge                                                                                   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="alembic_addnon_check">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Add non matching content from file</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="alembic_removenon_check">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text">
                <string>Remove non matching contnet from scene</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="alembic_sapcer">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="alembic_fit_check">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Fit Time Range</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="alembic_set_check">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Set Current Time to Start Frame</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="alembic_debug_check">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Debug</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="alembic_importnm_check">
           <property name="text">
            <string>Import Namespaces</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_import">
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
